sklearn/ensemble/forest.py

```diff
@@ -160,9 +160,9 @@ def apply(self, X):
         Parameters
         ----------
         X : array-like or sparse matrix, shape = [n_samples, n_features]
-            The input samples. Internally, it will be converted to
-            ``dtype=np.float32`` and if a sparse matrix is provided
-            to a sparse ``csr_matrix``.
+            The input samples. Internally, it's dtype will be converted to
```

Comment:
Thanks for catching that! I'll update in a bit when I'm back at a computer.

On Aug 27, 2016 4:42 PM, "Loïc Estève" <notifications@github.com> wrote:

> In sklearn/ensemble/forest.py
> <https://github.com/scikit-learn/scikit-learn/pull/6926#discussion_r76522723>
> :
>
> > @@ -160,9 +160,9 @@ def apply(self, X):
> >          Parameters
> >          ----------
> >          X : array-like or sparse matrix, shape = [n_samples, n_features]
> > -            The input samples. Internally, it will be converted to
> > -            ``dtype=np.float32`` and if a sparse matrix is provided
> > -            to a sparse ``csr_matrix``.
> > +            The input samples. Internally, it's dtype will be converted to
>
> Shouldn't it be "its dtype" (here and in every similar sentence below) ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/scikit-learn/scikit-learn/pull/6926/files/55cf091aa6130a7bea0cb554f6e50500013f10f0#r76522723>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGsSVaBDKRMJMW3Ae7Y7YJvsesCR14l_ks5qkK9OgaJpZM4I8Sk4>
> .
>